modules/apps/segments/segments-simulation-web/src/main/java/com/liferay/segments/simulation/web/internal/simulator/PortalPreferencesUserSegmentsEntrySimulator.java

```diff
@@ -85,11 +85,8 @@ public long[] getSimulatedSegmentsEntryIds(long userId) {
 			return new long[0];
 		}
 
-		Stream<String> stream = Arrays.stream(simulatedSegmentsEntryIds);
-
-		return stream.mapToLong(
-			Long::valueOf
-		).toArray();
+		return TransformUtil.transformToLongArray(
+			Arrays.asList(simulatedSegmentsEntryIds), Long::valueOf);
```

Comment:
@LisztLiszt Could you use `Long::parseLong` instead? The difference is, `Long::valueOf` returns `Long`, while `Long::parseLong` returns `long`. Here we convert a `String[]` to a `long[]`, so using the latter one avoids unnecessary unboxing.